sunpy/database/database.py

```diff
@@ -954,6 +955,13 @@ def redo(self, n=1):
         """
         self._command_manager.redo(n)  # pragma: no cover
 
+    def display_entries(self, columns=None, sort=False):
+        print (create_display_table(self, columns, sort))
+
+    def show_in_browser(self, columns=None, sort=False, **kwargs):
+        jsviewer = kwargs.get('jsviewer', True)
```

Comment:
just put it in the `def` line:
`def show_in_browser(self, columns=None, sort=False, jsviewer=True)`